src/methods/simlr/script.R

```diff
@@ -63,4 +63,5 @@ output <- anndata::AnnData(
   ),
   shape = input$shape
 )
+output$obs_names <- input$obs_names
```

Comment:
FYI -- I think I sometimes solve this by:

```suggestion
  obs = input$obs[,integer(0)],
  var = input$var[,integer(0)]
)